src/main/java/org/jabref/gui/groups/RenameGroupView.java

```diff
@@ -0,0 +1,85 @@
+package org.jabref.gui.groups;
+
+import java.util.Optional;
+
+import javafx.fxml.FXML;
+import javafx.scene.control.ButtonType;
+import javafx.scene.control.Label;
+import javafx.scene.control.TextField;
+import javafx.scene.layout.VBox;
+
+import org.jabref.gui.DialogService;
+import org.jabref.gui.util.BaseDialog;
+import org.jabref.model.database.BibDatabaseContext;
+import org.jabref.model.groups.AbstractGroup;
+
+import jakarta.inject.Inject;
+import org.jspecify.annotations.Nullable;
+
+public class RenameGroupView extends BaseDialog<AbstractGroup> {
+
+    @FXML
+    private TextField nameField;
+
+    @Inject
+    private DialogService dialogService;
+
+
+    private final BibDatabaseContext currentDatabase;
+    private final @Nullable AbstractGroup editedGroup;
+
+    public RenameGroupView(BibDatabaseContext currentDatabase,
+                           @Nullable AbstractGroup editedGroup) {
```

Comment:
What happens if this is null? - I think, this also also should not be null? - Please add JavaDoc if it can be null

Reply:
I removed Nulls. And it still works fine.